api/lib/domain/usecases/export-translations.js

```diff
@@ -21,18 +21,20 @@ export async function exportTranslations(stream, dependencies) {
       ]),
   );
 
-  const frenchChallenges = release.content.challenges
-    .filter((challenge) => challenge.locales.includes('fr'));
+  const localeToExtract = 'fr';
+
+  const frenchValidedChallenges = release.content.challenges
```

Comment:
vu qu'on sort la locale dans une variable dédiée, je propose de s'affranchir de la locale dans le nom de la variable : 
```suggestion
  const validedChallenges = release.content.challenges
```